packages/example/ios/Podfile

```diff
@@ -5,7 +5,7 @@ require Pod::Executable.execute_command('node', ['-p',
     {paths: [process.argv[1]]},
   )', __dir__]).strip
 
-platform :ios, min_ios_version_supported
+platform :ios, '14.0'
```

Comment:
Do we need to change it like this, could we change the `min_ios_version_supported` var instead?